Zend/Optimizer/zend_dump.c

```diff
@@ -49,6 +50,8 @@ void zend_dump_ht(HashTable *ht)
 
 void zend_dump_const(const zval *zv)
 {
+	zend_string *escaped_string;
```

Comment:
Please move the declaration to the given scope. You'll need to add a `;` -> `case IS_STRING:;` to make C happy.

Reply:
Done

Reply:
Nit: You can merge the declaration and the assignment.

Reply:
No problem